src/castra/middleware.clj

```diff
@@ -152,65 +107,29 @@
                   *request*       req
                   *session*       (atom (:session req))
                   *validate-only* (= "true" (get-in req [:headers "x-castra-validate-only"]))]
-          (let [h (headers req head {"Content-Type" "application/json;charset=utf-8"})
-                f #(do
-                     (csrf!)
```

Comment:
I removed the `csrf!` function because it turns out that it wasn't serving any purpose. See https://github.com/adzerk/castra/pull/6/commits/b3bb775a7a621d7a4de1bbf68a3c427849b747dc